index.html

```diff
@@ -1,29 +1,17 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <title>Checklist</title>
-        <link rel="stylesheet" href="main.css">
+		<title>MarkDownTextArea</title>
+		<link rel="stylesheet" href="main.css">
+        <link rel="stylesheet" type="text/css" href="https://yarnpkg.com/en/package/normalize.css">
```

Comment:
Why are we using this library?

Reply:
I used this library to test the styling it was supposed to be removed before pushing.